pre-pages/pgql-2.0-spec.md

```diff
@@ -3175,25 +3751,34 @@ For example, in the following query the first solution is pruned from the result
 {% include image.html file="example_graphs/financial_transactions.png" %}
 
 ```sql
+--PGQL
 SELECT n.name
 FROM MATCH (n:Person) ON financial_transactions
 ORDER BY n.name
 OFFSET 1
-FETCH FIRST 2 ROWS ONLY
+FETCH FIRST 1 ROWS ONLY
+--SQL
+SELECT name 
+FROM GRAPH_TABLE(financial_transactions 
+  MATCH (n IS Person) 
+  COLUMNS(n.name)) 
+ORDER BY name 
+OFFSET 1 
+FETCH FIRST 1 ROWS ONLY
```

Comment:
description above says FETCH FIRST 2 ROWS ONLY but here we have 1 ROWS
so it's still not entirely consistent.

Reply:
I had to reduce the number of results to show the effect in the resultset. the query used matched 3 Persons but the resultset was wrong(showed the company name as well), I fixed the resultset and reduced the number. I will update the description.